src/components/Index/VotingContent.svelte

```diff
@@ -11,8 +11,8 @@
 <div>
 	<h3 class="fluid-heading-04 mb-4">{latestVotings.length} ผลการลงมติล่าสุด</h3>
 	<Carousel>
-		{#each latestVotings as { voting, highlightedVoteByGroups } (voting.id)}
-			<VoteCard class="keen-slider__slide" {voting} {highlightedVoteByGroups} />
+		{#each latestVotings as voting}
+			<VoteCard class="keen-slider__slide" {voting} />
```

Comment:
VSCode show type error here. As defined in line 8, `voting` variable contain every props from `VoteCard` component, not just the `voting` prop.

You can spread properties in object into component props like:

```svelte
<VoteCard class="keen-slider__slide" {...voting} />
```